app/models/hipPenaltyDetails/lateSubmission/LSPDetails.scala

```diff
@@ -43,18 +43,29 @@ case class LSPDetails(
 )
 
 object LSPDetails {
+
+  // TODO: DL-17378 - when ETMP have merged their fix we can revert to just checking for a String
+  private val penaltyOrderOptionalStringReads: Reads[Option[String]] =
+    Reads[Option[String]] {
+      case JsString(s) => JsSuccess(Some(s))
+      case JsNumber(n) => JsSuccess(Some(n.toInt.toString))
+      case JsNull      => JsSuccess(None)
+      case _           => JsError("penaltyOrder must be a JsString or JsNumber")
+    }
+
   implicit val reads: Reads[LSPDetails] = (json: JsValue) =>
     for {
       penaltyNumber <- (json \ "penaltyNumber").validate[String]
-      penaltyOrder <- (json \ "penaltyOrder").validateOpt[String]
+      penaltyOrder  <- (json \ "penaltyOrder").validate(penaltyOrderOptionalStringReads)
```

Comment:
PenaltyOrder is a 2 digit number from "01" to "99". However, "01" - "09" are being returned as strings and 10 - 99 are being returned as numbers